src/.env.sample

```diff
@@ -180,5 +180,8 @@ OBSERVATION_DEEP_LINK_REGEX= "^https:\/\/dev\.elevate-ml\.shikshalokam\.org\/vie
 # Mongo DB
 MONGODB_URL=mongodb://localhost:27017/elevate-project
 
+# detault data manager roles 
+DEFAULT_DATA_MANAGERS="program_manager,program_designer"
+
```

Comment:
@adithyadinesh0412 program manager is used for program create  i think here they want data manager please check the default role with Niveditha

Reply:
currently these are the roles in Diksha and EP doesn't have this capability yet @priyanka-TL . @priyanka-TL